packages/ui-components/src/components/Chart/Tooltip.js

```diff
@@ -120,9 +120,12 @@ const SingleValueTooltip = ({ valueType, payload, periodGranularity, labelType }
 export const Tooltip = props => {
   const { payload, active, presentationOptions } = props;
 
-  const data = payload || []; // This is to hancle when recharts overrides the payload as null
+  const data = payload || []; // This is to handle when recharts overrides the payload as null
   const filteredPayload = data.filter(({ value }) => value !== undefined);
 
+  // Todo: lookup
+  // const filteredPayload = data.filter(({ value }) => ![null, undefined].includes(value));
```

Comment:
👀

Reply:
Thanks for checking this. I was initially struggling to understand what this mean but I finally figured it out and realised that I need to keep this change.